testing/regress/ecl/keydiff1.ecl

```diff
@@ -18,6 +18,7 @@
 import $.setup;
 prefix := setup.Files(false, false).QueryFilePrefix;
 
+#onwarning (30003,ignore);
```

Comment:
perhaps add a brief comment saying what warning this is and why it is ignoring here.